### DISCUSSION TITLE
new version

### DISCUSSION THREAD

**Discussion by q16marvin** [Lissy93/dashy#1169] [General]
Hi, 

first thx for such awesome tool :) looks great!

currently release 2.1.1 is quit old. is there any chance to got "newer" version or only via "Deploy from Source?" 

Thx!

Erik

**Comment by CrazyWolf13** [comment]
> Hi,
> 
> first thx for such awesome tool :) looks great!
> 
> currently release 2.1.1 is quit old. is there any chance to got "newer" version or only via "Deploy from Source?"
> 
> Thx!
> 
> Erik

On docker there is a 12 day old image.

**Reply by shrippen** [reply]
Which is still 2.1.1

**Reply by CrazyWolf13** [reply]
I guess it's a slightly updated 2.1.1, just try it, I'm sure once lissy93 will have more times new releases will be more frequent.

But I think at the moment we can only wait.
